lib/model/QubitRepository.php

```diff
@@ -124,14 +124,32 @@ public function save($connection = null)
     return $this;
   }
 
+  public function getRelatedInformationObjectIds()
+  {
+    $sql = "SELECT id FROM ". QubitInformationObject::TABLE_NAME ." WHERE repository_id=:repository_id";
+
+    $results = QubitPdo::fetchAll($sql, array(':repository_id' => $this->id));
```

Comment:
If you don't need the rows returned as objects, memory can be saved here by using `$results = QubitPdo::fetchAll($sql, $params, array('fetchMode' => PDO::FETCH_ASSOC));`.

Reply:
Good call!